velox/docs/functions/spark/decimal.rst

```diff
@@ -0,0 +1,13 @@
+=================
+Decimal functions
+=================
+
+Spark decimal functions can be used in decimal value computation.
+
+Value functions
```

Comment:
What does "Value" mean here? Do we need this section at all? What other functions are you planning to add?

Reply:
We will add more decimal functions in spark sql, such as make_decimal, check_overflow, round_decimal.

Reply:
> We will add more decimal functions in spark sql, such as make_decimal, check_overflow, round_decimal.

is there a GitHub issue that lists all these functions already? If not, let's create one.

> I have a concern, these functions are used internally

What do you mean by "internally"? Are these used in Gluten? If so, then from Velox perspective that usage is "external".

Reply:
What I mean is that these methods are not used by users, but spark itself will use these methods when constructing plans. For example, the unscaled_value function is used to optimize the calculation of short decimal and directly convert it to long for calculation.

Reply:
Makes sense. However, from Velox's perspective, these are externally visible / used functions. Hence, we should document them.